tests/tests_combination.yml

```diff
@@ -130,15 +141,25 @@
       failed_when: not __result.stat.exists
 
     - name: Check the input call with tag={{ __test_tag }}
-      command: /bin/grep ' *input(type="imfile" file="\/var\/log\/inputdirectory\/\*.log" tag="{{ __test_tag }}"' '{{ __test_inputfiles_conf }}'
+      command: "/bin/grep
```

Comment:
with `command` it is better to use `>-` which will generate a single line string with each newline replaced with a single space